docs/chain/fullnode.md

```diff
@@ -116,7 +116,7 @@ Once you have the image, you can spin up the container.
 Make sure to choose whether you want to start a full node for peregrine or spiritnet by selected the correct runtime and chain.
 
 ```bash
-docker run data:/data kiltprotocol/kilt-node:latest \
+docker run -v ~/data:/data kiltprotocol/kilt-node:latest \
```

Comment:
You are right that the `-v` was missing, but prefixing`data` with `~/` will cause docker to mount the directory `data` from the home directory of the running user instead of the named volume `data` (which will be rooted somewhere at `/usr/foo` for docker or `~/.local/share/containers/storage/volumes/data/` in case you are using podman.

Thinking it through it should be `docker run -v kilt-node-data:/data kiltprotocol/kilt-node:latest` since the named volumes are all represented in the host mount namespace by that name, so choosing a recognizable and less generic name would make sense.